source/develop/release-management/features/virt/high-performance-vm.html.md

```diff
@@ -0,0 +1,179 @@
+---
+title: High Performance VM 
+category: feature
+authors: SharonG
+feature_name: High Performance VM
+feature_modules: engine
+feature_status: Planning
+---
+
+# High Performance VM
+
+## Summary
+
+Support a new type of VM in oVirt destined for running a VM with highest possible performance and performance metrics as close to bare metal as possible. 
+
+For setting a VM as High Performance, a new VM profile type named "High Performance" will be added in addition to existing "Server", "Desktop" types.  By choosing This new High Performance VM type,  the VM will be pre-configured with a set of suggested and recommended configuration settings for reaching the best efficiency.
+
+## Owner
+*   Name: Sharon Gratch (sgratch)
+*   Email: <sgratch@redhat.com>
+
+## Benefit to oVirt
+
+Before this feature was implemented, configure a VM to run with high performance workloads was not an easy straightforward mission to do and required the user to manually set the VM as such by going over all settings and check what is relevant and how to configure it. 
+
+Furthermore, few required features essential for improving VM performance were not supported at all by oVirt (for example, using huge pages or I/O Thread pinning) and new features as Headless VMs can now be leveraged to suggest one solution of the best recommended configuration according to VM usage requirements.
+
+This feature introduces a simple to manage solution for running a new/existing VM as High Performance via WebAdmin or REST api, while the user can still keep an option to manually change or ignore the suggested configuration for his own tuning and requirements. 
+
+## Usage
+
+ -  Setting a VM to run as a High Performance VM can be done to a new or existing VM via UI by selecting the "High Performance" type in the "Optimized for" pull down menu field displayed in VM new/edit dialog. 
+This new VM type is added in addition to already existed VM types: "Server", "Desktop". 
+ 
+ - In case of changing the VM type for a running VM, part of the required configuration changes will require a VM restart after saving. In case of changing the VM type for a new/existed VM, part of the required/suggested configuration may require manually changes of the VM's cluster configuration prior to setting the VM as High-performance, and part of the required/suggested configuration may require changing of the specific pinned host configuration prior to setting the VM as High-performance.
+
+ -  A High Performance Template or Pool can be created in the same way as a VM. In case the user wants to create or edit a VM to become High Performance, he can also choose a Template or a Pool which are configured as High Performance type and "inherit" this property for that specific VM.
```

Comment:
Partially correct. In case of stateless VMs configured to run with latest template version, editing VM type of the template latest version will trigger all existed stateless VMs to be changed accordingly (after restart). But I will re-phrase this.. :)